linkerd.io/layouts/shortcodes/trylpt.html

```diff
@@ -0,0 +1 @@
+{{ partial "admonition.html"    (dict        "type" "info"        "icon" "info-circle"        "id" (.Get "id")        "title" "Linkerd Production Toolkit"        "content" "This page shows instructions for example configurations. For recommended configurations, continuous monitoring, and automated management of Linkerd connect with a <a href=\"/support-training/\">Linkerd Provider</a> and explore the <a href=\"https://buoyant.io/linkerd/getting-started\">Linkerd Production Toolkit by Buoyant</a>."    )}}
```

Comment:
```suggestion
{{ partial "admonition.html"    (dict        "type" "info"        "icon" "info-circle"        "id" (.Get "id")        "title" "Linkerd Production Tip"        "content" "This page shows instructions for example configurations. For recommended production configurations, continuous monitoring, and automated management of Linkerd explore the <a href=\"https://buoyant.io/linkerd/getting-started\">Linkerd Production Toolkit by Buoyant</a>."    )}}
```
Example configuration or dev cluster configuration? 

I think I'd not mention vendors in the doc and only point to the production tool kit for production use cases.

Reply:
How about: This page shows instructions for a quickstart, non-production install. For production configurations including continuous monitoring and vulnerability alerts, we suggest connecting with a <a href=\"/support-training/\">Linkerd provider</a> or using the <a href=\"https://buoyant.io/linkerd/getting-started\">Linkerd Production Toolkit</a> by Buoyant.